### PR TITLE
Add TagID filter on GET /Users request

### PR DESCRIPTION
**Pull request by roxiciobotea210**
The filter is used in calls form the frontend but it was not implemented on the backend side.

### CODE REVIEW

File: src/server/rest/v1/service/UserService.ts [hunk 1]
```diff
@@ -1316,6 +1316,7 @@ export default class UserService {
         issuer: filteredRequest.Issuer,
         siteIDs: (filteredRequest.SiteID ? filteredRequest.SiteID.split('|') : null),
         userIDs: (filteredRequest.UserID ? filteredRequest.UserID.split('|') : null),
+        tagID: (filteredRequest.TagID ? filteredRequest.TagID : null),
```
Comment (jerome-benoit) [inline src/server/rest/v1/service/UserService.ts:1319]: Does the filter allow several tags ID? 
If yes, you have to use an array.

**Review by jerome-benoit — COMMENTED** [review]
One question.

File: src/server/rest/v1/service/UserService.ts (hunk 1 above)
Comment (roxiciobotea210) [reply src/server/rest/v1/service/UserService.ts:1319]: No, at the moment the filter only supports on TagID.

Comment (roxiciobotea210) [reply src/server/rest/v1/service/UserService.ts:1319]: Thanks for mentioning, it does need to support several tag IDs! Working on the change now.

**Comment by roxiciobotea210** [comment]
@jerome-benoit Done.

**Review by jerome-benoit — APPROVED** [review]

**Pull request merged**